### ISSUE TITLE
Support for older SSE2 CPUs

### ISSUE THREAD

**Issue by Stepman123** [weolar/xpchrome#3]
At the moment, there are already two projects with support for SSE2 in modern versions, these are Supermium and Thorium.

**Comment by andika207** [comment]
if WinXP is your daily driver system you should ditch that piece of crap and pick a decent CPU for it
it's makes no sense to request a modern browser while keeping obsolete hardware

**Comment by sven204** [comment]
If a modern Web browser becomes available for such systems, that will obviously bring them one step further from obsolescence.

**Comment by tbone2k-git** [comment]
@andika207 andika207
> if WinXP is your daily driver system you should ditch that piece of crap and pick a decent CPU for it it's makes no sense to request a modern browser while keeping obsolete hardware

So the new chromium makes sense for old WindowsXP, but not for old hardware? Interesting point of view, to me both are obsolete, kind of, but it's still fun using old OS and hardware, so why not have both! o)

I would go even further and request a Non-SSE2 version as well, so it can run on my old Pentium3 machines. The Pentium3 is not necessarily slower than the early Pentium4 with SSE2. I have several Pentium3 laptops and machines in mint condition, it would be cool to surf the web with them! o)

**Comment by Stepman123** [comment]
> modern browser

This browser ceased to be modern at the moment when you installed it on XP.

**Comment by andika207** [comment]
> So the new chromium makes sense for old WindowsXP, but not for old hardware? 

of course, we use WinXP because we don't like the W10/11 junk however IE8 doesn't work anymore so we need a modern browser for it which requires a decent hardware, from around 2010 is somewhat better than that crap from the early 2000's

**Comment by andika207** [comment]
> I would go even further and request a Non-SSE2 version as well, so it can run on my old Pentium3 machines. 

as expected you want a modern browser to play around a few minutes upon crap hardware but you actually have zero interest on WinXP because it's obsolete for you

**Comment by tbone2k-git** [comment]
@andika207 
I set up quite a lot of machines running XP IntegralEdition this year, some were AthlonXP with NonSSE2 cpus, some were quad core machines from 2010+ (what you describe as "decent hardware" o). I am using such piece of kit right now, it's still capapable of most work loads. I disabled all the Win10 junk by a custom "debloater" script, I can't stand the annoying stuff either. So, I really think we are in the same boat, no point to argue! o)

I'm curious, do you actually use the 32bit version of XP as daily driver?

@weolar 
Talking about having a Non-SSE2 version as well, isn't it just a flag when compiling? So maybe it's not that much additional effort? You know better than me - it's been some time since I had contact with gcc / visual studio and stuff.

Thank you all! o)

**Comment by andika207** [comment]
> I set up quite a lot of machines running XP IntegralEdition this year,

now way will I use this crap unattended XP black edition with another skin.... anyway it fails to install on intel's 8 series chipset setups

**Comment by andika207** [comment]
>I'm curious, do you actually use the 32bit version of XP as daily driver?

of course and I only miss an up to date browser because 360chrome is getting somewhat obsolete on some sites (github for example)
Win7 is my testbed OS I mostly use it for playing some games or surfing faster with Supermium.

 I'm not the [typical moron]( https://github.com/win32ss/supermium/issues/29#issuecomment-1786227998) who gets bored with W11 and installs WinXP on a virtual machine and then goes to reddit ask others what software utilities are necessary for WinXP LOL

**Comment by tbone2k-git** [comment]
Up until now I'm quite satisfied with the IntegralEdition, it installs flawlessly with nearly all updates built in. You can skip the extra themes, the unattended installer allows for some interventions/options and once it's done, you're basically ready to roll. If you have specific hardware, like the AthlonXP machines I was talking about, there is a patcher you can run upfront. It allows to remove/replace SSE2+ components from the ISO or let's you add specific storage/usb3 etc. drivers. This package is not something you put together on a rainy sunday afternoon, not even speaking about major updates to all this once in a while.

So, can you tell why the IntegralEdition is "crap" exactly? Is there a better alternative?

**Comment by andika207** [comment]
> So, can you tell why the IntegralEdition is "crap" exactly?

didn't somebody ask for the source code https://github.com/weolar/xpchrome/issues/1#issue-1996001001 of this web browser ?
do you trust on a modified XP ISO without the source code being made available to the public ?

also the bundled software becomes obsolete overtime whether you install it or not, Mypal68 to name one is a buggy browser that should never be offered there in the options selector menu.  LOL

**Comment by andika207** [comment]
>Is there a better alternative?

grab a genuine XP ISO in your native language and slipstream your necessary drivers with Patch Integrator and then get it up to date with Legacy Update https://github.com/kirb/LegacyUpdate

**Comment by tbone2k-git** [comment]
Did you check source code of updates being pushed by LegacyUpdate to your computer?
Did you verify the binaries for download were actually made from the source in the repo?
What did you use before LegacyUpdate? It's been around 1 year only?!

My impression is, you don't really have serious arguments against IntegralEdition, am I right?

I mean c'mon.. the pal-browser, just don't "check" it when running the setup! And regarding potential security issues, well yes maybe, but you have this problems with every other release and LegacyUpdate as well, don't you?

**Comment by andika207** [comment]
> Did you check source code of updates being pushed by LegacyUpdate to your computer? 

LOL source code of updates ? @kirb

I won't convince you, good luck with your modified **XP Black Edition** ISO

**Comment by tbone2k-git** [comment]
Please, let's have a fact based conversation! o)

I was mentioning source code of updates, because I was thinking that LegacyUpdate would provide additional fixes and security patches over what Microsoft offered back in the days, but it seems this is not the case. I find this a bit unfortunate to be honest, since there is no advantage over a selfmade ISO with all the patches integrated, correct? o(

You really can convince me to ditch IntegralEdition, but please provide some pro/con facts, like:
LegacyUpdate is using un-encrypted communication during the update process. It is collecting system information. It will not give a more recent update status than selfmade ISOs. All the POSReady 2009 fixes are missing. No backported drivers from Win7 + 8 included for SSD, SATA USB3. Also no patches to include features / updates from Win2k3 Server. LegacyUpdate is useless if Microsoft removes the XP updates from their servers.

What's the actual fact based con-list for IntegralEdition?

Thank you in advance! o)
